Makefile

```diff
@@ -3,7 +3,7 @@ SDIR=src
 TDIR=pytests
 # compilers
 TSC=./node_modules/typescript/bin/tsc
-TSCFLAGS=-t ES3 --sourceMap --module commonjs --noImplicitAny
```

Comment:
Also, if it fails to compile, it will give you a nice error message telling you the issue... so you should track it down and fix it by adding a type annotation.